python/mxnet/rnn/rnn_cell.py

```diff
@@ -788,6 +788,12 @@ def unpack_weights(self, args):
     def pack_weights(self, args):
         return _cells_pack_weights(self._cells, args)
 
+    def reset(self):
+        super(SequentialRNNCell, self).reset()
+        if hasattr(self, '_cells'):
```

Comment:
SequentialRNNCell's constructor will call BaseRNNCell's constructor first before it initializes `self._cells`, BaseRNNCell's constructor will call `self.reset()`. So at first time `hasattr(self, '_cells')` should be false. @szha